CHANGES.rst

```diff
@@ -17,9 +17,15 @@ astropy.coordinates
 ^^^^^^^^^^^^^^^^^^^
 
 - The ``Distance`` object now accepts ``parallax`` as a keyword in the
-  initializer, and supports retrieving a parallax (as an ``Angle``) via 
+  initializer, and supports retrieving a parallax (as an ``Angle``) via
   the ``.parallax`` attributes. [#6855]
 
+- The coordinate frame classes (subclasses of ``BaseCoordinateFrame``) now
+  always have ``.velocity``, ``.proper_motion``, and ``.radial_velocity``
+  properties that provide shorthands to the full-space Cartesian velocity as
+  a ``CartesianDifferential``, the 2D proper motion as a ``Quantity``, and the
+  radial or line-of-sigh velocity as a ``Quantity``. [#6869]
```

Comment:
`line-of-sigh` sounds like a lot of code I write regularly... But I think you meant `line-of-sight`.